src/neuroconv/tools/spikeinterface/spikeinterface.py

```diff
@@ -348,10 +348,10 @@ def add_electrodes(
         if "group" in data_to_add:
             group_name_array = data_to_add["group"]["data"].astype("str", copy=False)
         else:
-            default_group_name = "default_group"
-            group_name_array = np.full_like(channel_name_array, fill_value=default_group_name)
+            default_group_name = "ElectrodeGroup"
```

Comment:
Changing the default as discussed in https://github.com/catalystneuro/neuroconv/pull/70#issuecomment-1213239985 . 

Moreover, changing to `np.full` from `np.full_like` because using numpy array with strings is always tricky. Numpy implicitly defines a character size as the type and then cuts the size of your strings silently if they are longer than than the defined type (in this case I was confused for a while why a group called "Electro" was being created (turned out it was np.full_like cutting the strings).